### PR TITLE
Add robust median to gopher filter

### PR DESCRIPTION
**Pull request by KennethEnevoldsen**
This pull request adds a robust median function to the gopher filter, which would otherwise fails on empty docs such as "" or " ".

Sorry for the multiple commit wanted to add @TTTTao725 as a co-author as he found the [bug](https://github.com/centre-for-humanities-computing/danish-foundation-models/issues/209).

### CODE REVIEW

**Comment by KennethEnevoldsen** [comment]
Updated in accordance with failing tests

**Comment by soldni** [comment]
tnx!! added a comment.

File: python/dolma/taggers/gopher.py
Comment (KennethEnevoldsen) [reply python/dolma/taggers/gopher.py:76]: It seems like it starts out as a False, so tried to match the existing pattern. However the median can be undefined (empty list) but multiple value could represent that (np.nan, 0, False). I would probably go for np.nan or None if that is valid?

**Comment by soldni** [comment]
for the python style error, you should be able to run `make style` to fix any issue :)

**Comment by KennethEnevoldsen** [comment]
Hi @soldni fixed the style errors!

**Comment by KennethEnevoldsen** [comment]
@soldni it seems like this is awaiting an approval for the tests to be run

**Comment by soldni** [comment]
Hey @KennethEnevoldsen! I've made a few more changes:

- I did find some issues in how `black` and `isort` were configured, which was causing issues in test failing on GitHub but not locally. Sorry for the hassle it caused! 
- I changed the median such that it always returns a float. 

I've approved this PR, lmk if it looks good to you too before I merge!

**Review by soldni — APPROVED** [review]

**Comment by KennethEnevoldsen** [comment]
Thanks for taking care of that. Everything looks good to me

**Pull request merged**
